### PR TITLE
Altom-week-6

### PR DESCRIPTION
**Pull request by AltomHussain**
# Your Details

Your Name:
Your City:
Your Slack Name:

# Homework Details

Module:
Week:

### CODE REVIEW

File: week-6/Homework/mandatory/1-alarmclock/alarmclock.js [hunk 1]
```diff
@@ -1,7 +1,32 @@
...
+  document.getElementById("alarmSet").value = "";
+  let count = document.getElementById("timeRemaining");
 
-function setAlarm() { }
+  if (timer) {
+    clearInterval(timer);
+    num < 10
+      ? (count.innerText = "Time Remaining: 00:0" + num)
+      : (count.innerText = "Time Remaining: 00:" + num);
+  }
```
Comment (Abdoulrazack95) [inline week-6/Homework/mandatory/1-alarmclock/alarmclock.js:18]: () for the conditional ternary  is missing here Altom. It needs to be fixed

File: week-6/Homework/mandatory/1-alarmclock/alarmclock.js [hunk 2]
```diff
@@ -1,7 +1,32 @@
...
+    if (num > 0) num--;
+    num < 10
+      ? (count.innerText = "Time Remaining: 00:0" + num)
+      : (count.innerText = "Time Remaining: 00:" + num);
+    if (num === 0) {
+      clearInterval(timer);
+      playAlarm();
+    }
+  }, 1000);
+}
```
Comment (Abdoulrazack95) [inline week-6/Homework/mandatory/1-alarmclock/alarmclock.js:30]: Well done Altom.

File: week-6/Homework/mandatory/2-quotegenerator/quotes.js [hunk 3]
```diff
@@ -19,7 +32,7 @@
 function pickFromArray(choices) {
   return choices[Math.floor(Math.random() * choices.length)];
 }
-
+window.onload = quoteButton;
 // A list of quotes you can use in your app.
 // Feel free to edit them, and to add your own favourites.
 const quotes = [
```
Comment (Abdoulrazack95) [inline week-6/Homework/mandatory/2-quotegenerator/quotes.js:38]: Excellent work Altom.

File: week-6/Homework/mandatory/3-slideshow/slideshow.js [hunk 4]
```diff
@@ -1 +1,54 @@
...
+  "https://image.shutterstock.com/image-photo/bright-spring-view-cameo-island-260nw-1048185397.jpg",
+  "https://media3.s-nbcnews.com/j/newscms/2019_41/3047866/191010-japan-stalker-mc-1121_06b4c20bbf96a51dc8663f334404a899.fit-760w.JPG",
+  "https://i.dailymail.co.uk/1s/2020/01/07/00/23077356-7858657-image-a-88_1578357487105.jpg",
+  "https://cdn.pixabay.com/photo/2015/04/23/22/00/tree-736885__340.jpg",
+  "https://www.gettyimages.com/gi-resources/images/500px/983794168.jpg",
+  "https://www.elyssamcgregor.com/wp-content/uploads/2018/11/what-is-a-gif.jpg",
+];
+changeHTML(images);
+// window.onload = forward;
+console.log(images.length);
```
Comment (Abdoulrazack95) [inline week-6/Homework/mandatory/3-slideshow/slideshow.js:54]: Excellent work Altom

File: week-6/Homework/mandatory/1-alarmclock/alarmclock.js (hunk 1 above)
Comment (zoltan-gal) [reply week-6/Homework/mandatory/1-alarmclock/alarmclock.js:18]: I think braces are optional. But might be a good practice to put them.

**Pull request closed without merging**